db/migrate/20220603095337_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string      :title
+      t.text        :text
+      t.integer     :comments_count, default: 0
+      t.references :user, foreign_key: true
```

Comment:
- [ ] Let's add `likes_counter` in this migration to match [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

Reply:
@uwadonat 
Good catch, totally missed that.
Cheers,